kernel/cPrimitives.mli

```diff
@@ -105,8 +132,12 @@ val op_or_type_to_string : op_or_type -> string
 
 val parse_op_or_type : ?loc:Loc.t -> string -> op_or_type
 
-type ind_or_type =
-  | PITT_ind : 'a prim_ind * 'a -> ind_or_type
-  | PITT_type : prim_type -> ind_or_type
+val univs : t -> Univ.AUContext.t
+
+val types : t -> Constr.rel_context * ind_or_type list list
+(** Parameters * Reduction relevant arguments and output type
+
+    TODO reify universes in ind_or_type (currently polymorphic types
```

Comment:
@SkySkimmer should this be addressed now?

Reply:
I think this can be left for future improvement. Perhaps XXX would convey that better than TODO.